README.md

```diff
@@ -61,6 +28,11 @@ To build, install, and test the project from the command line:
     $ ./gradlew :WordPress:testWordPressVanillaDebugUnitTest         # assemble, install and run unit tests
     $ ./gradlew :WordPress:connectedWordPressVanillaDebugAndroidTest # assemble, install and run Android tests
 
+## Running the app ##
+
+You can use your own WordPress site for developing and testing the app. If you don't have one, you can create a temporary test site for free at https://jurassic.ninja/.
+On the app start up screen choose "Enter your existing site address" and enter the URL of your site and your credentials.
```

Comment:
np: ChatGPT suggested adding a comma after "screen": `On the app start up screen, choose...`.

Reply:
Good catch Irfan. Fixed with 2557cba29d2161ccfd17a63ba18b17bbc0776304